### ISSUE TITLE
increase decimal places on fold80 column

### ISSUE THREAD

**Issue by chrispyatt** [eastgenomics/eggd_MultiQC_configs#7]
We exclude samples from CNV calling if fold80>=1.5, but the report displays rounded figures so 1.5 may mean 1.49 (not excluded) or 1.51 (excluded). We currently have to delve into the raw data file to find the exact number but we should just display more decimal places (3 should be enough to avoid this issue most of the time?).

We should be able to add something like the code below to the config to improve this:

custom_plot_config:
  picard_hsmetrics_table:
    FOLD_80_BASE_PENALTY:
      format: "{:,.3f}"

**Comment by Emma-MacK** [comment]
Looking into this it looks like the version of MultiQC we currently use does not have the functionality to change specific columns in modules. We use a docker of v1.11 and this function was added in v1.13. 

```
We use docker 1.11 of multiQC: 001_Reference/assets/multiQC/seglh_multiqc_v1.11.tar.gz
documentation for v1.11 is different that the documentation as it stands, lacking the config for an entire table and Config for a specific column sections. 
picard is a module containing HSMetrics 


key changes between versions:

1.16 Give config custom_plot_config priority over column-specific settings set by modules
1.14 HsMetrics: Allow custom columns in General Stats too,
	   with HsMetrics_genstats_table_cols and HsMetrics_genstats_table_cols_hidden
1.13 Added new config option custom_table_header_config to override any config for any table header


since we dont have version 1.13 we cannot use custom_table_header_config for tables other that general_stats
since we dont have 1.16 custom_plot_config does not have priority for the picard module over set column specific settings 

recommended action: update the docker we have.
in DNAnexus it says that the docker was modified Apr 27 2023, 5:48 PM
```

Is updating the docker possible?

**Comment by chrispyatt** [comment]
fixed by #11 #12 #13 #14
